src/Microsoft.Azure.WebJobs.Extensions.OpenApi.Core/Visitors/ByteEnumTypeVisitor.cs

```diff
@@ -0,0 +1,129 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Abstractions;
+using Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Attributes;
+using Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Extensions;
+using Microsoft.OpenApi.Any;
+using Microsoft.OpenApi.Models;
+using Newtonsoft.Json.Converters;
+using Newtonsoft.Json.Serialization;
+
+namespace Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Visitors
+{
+
+    /// <summary>
+    /// This represents the type visitor for <see cref="byte"/> type enum.
+    /// </summary>
+    public class ByteEnumTypeVisitor : TypeVisitor
+    {
+        /// <inheritdoc />
+        public ByteEnumTypeVisitor(VisitorCollection visitorCollection)
+            : base(visitorCollection)
+        {
+        }
+
+        /// <inheritdoc />
+        public override bool IsVisitable(Type type)
+        {
+    
+            var isVisitable = this.IsVisitable(type, TypeCode.Byte) &&
+                              type.IsUnflaggedEnumType() &&
+                              !type.HasJsonConverterAttribute<StringEnumConverter>() &&
+                              Enum.GetUnderlyingType(type) == typeof(byte)
+                              ;
+
+            return isVisitable;
+        }
+
+        /// <inheritdoc />
+        public override void Visit(IAcceptor acceptor, KeyValuePair<string, Type> type, NamingStrategy namingStrategy, params Attribute[] attributes)
+        {
+            var name = type.Key;
+
+            var instance = acceptor as OpenApiSchemaAcceptor;
+            if (instance.IsNullOrDefault())
+            {
+                return;
+            }
+
+            // Adds enum values to the schema.
+            var enums = type.Value.ToOpenApiIntegerCollection();
```

Comment:
We need `ToOpenApiByteCollection()` method like `ToOpenApiInt16Collection()`

Reply:
Done.